transfers/transfer.py

```diff
@@ -296,7 +300,7 @@ def start_transfer(ss_url, ts_location_uuid, ts_path, depth, am_url, user_name,
         return None
     if not response.ok or resp_json.get('error'):
         LOGGER.error('Unable to start transfer.')
-        LOGGER.debug('Response: %s', resp_json)
+        LOGGER.error('Response: %s', resp_json)
```

Comment:
We should probably have just one single log debug stream in development environments, but we can look at that later! Thanks!